src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -71,15 +106,27 @@ export class CardKebabMenuButton extends React.Component<CardKebabMenuButtonProp
         // todo
     };
 
-    private copyFailureDetails = (): void => {
-        // todo
+    private copyFailureDetails = (event: React.MouseEvent<any>): void => {
+        const onClick = this.props.deps.detailsViewActionMessageCreator.copyIssueDetailsClicked;
```

Comment:
Why are we checking `detailsViewActionMessageCreator.copyIssueDetailsClicked` is not null here? is that even possible?